baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/LoggerEnclosingClass.java

```diff
@@ -83,8 +83,16 @@ public Description matchVariable(VariableTree tree, VisitorState state) {
         return buildDescription(classArgument)
                 .addFix(fix.replace(
                         classArgument,
-                        SuggestedFixes.prettyType(state, fix, enclosingClassSymbol.type) + ".class")
+                        enclosingClassSymbol.name + ".class")
```

Comment:
Simple name is sufficient here because the code we're updating is nested within the type -- the previous approach using prettyType wasn't necessary and included type parameters.